### PR TITLE
Fix the library to work with .NET 5, and fix issue #20 and issue #23

### PR DESCRIPTION
**Pull request by faith-ie**

### CODE REVIEW

**Comment by velddev** [comment]
Few points before I will review it:
- Please solve the spacing issues
- Please state what actually fixed in the PR title.

**Comment by faith-ie** [comment]
1) unless you're mentioning a different spacing issue, I shiouldve pushed a fix already after  I made this PR
2) fixed

**Comment by velddev** [comment]
It's hard for me to see what changed here: 
image

**Comment by faith-ie** [comment]
i believe i might have changed the spacing issue, if it still is there, there is nothing I can really do as it's codemaid's fault that the spacing is weird.

File: DiscordBotsList.Api/Internal/Bot.cs [hunk 1]
```diff
@@ -11,53 +11,41 @@ public class Bot : Entity, IDblBot
     {
         internal DiscordBotListApi api;
 
-        [JsonPropertyName("lib")]
-        public string library { get; set; }
+        [JsonPropertyName("lib")] public string library { get; set; }
```
Comment (velddev) [inline DiscordBotsList.Api/Internal/Bot.cs:14]: Could we not put attributes inline? It'll cause some wonkyness if there's multiple added, and it reads a little weird.

**Review by velddev — APPROVED** [review]

**Comment by velddev** [comment]
Overall, this PR is rather massive for what it does, I would recommend not to mix a full-project reformat with 5 property changes as 90% of the PR review was me looking at code that was the same.

**Pull request merged**